joy2twist/launch/joy2twist.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
+  <arg name="docker" default="false" /> 
+
+  <group if="$(arg docker)">
+    <rosparam file="/ros_ws/src/joy2twist/joy2twist/config/joy2twist.yaml"/>
+  </group>
+
+  <rosparam unless="$(arg docker)" file="$(find joy2twist)/config/joy2twist.yaml"/>
```

Comment:
is docker argument really necessary? We may load the same file, but in docker the one loaded in volume will be used, as it will overwrite old one. `$(find joy2twist)` probably leads to install directory, so it may be necessary to take it into account.

If you think that docker argument should stay, then it will be better to move `if` into rosparam instead of `group` - having two ways of writing conditions in one file isn't that nice.

Reply:
As I remember @pawelirh tested that and for some reason it did not work.

Reply:
I couldn't find a solution to this problem.
I'd stick with docker arg. Anyway, I corrected the launch file as you suggested. :heavy_check_mark: